integration_test/third_party_apps_data/applications/flink/debian_ubuntu/install

```diff
@@ -1,7 +1,12 @@
 set -e
 
 sudo apt-get update
-sudo apt-get install -y curl default-jre
+source /etc/os-release
+if [ "${VERSION_CODENAME}" = "lunar" ]; then
+  sudo apt-get install -y curl openjdk-11-jre
```

Comment:
Is this because `default-jre` on Ubuntu 23.04 pulls in `openjdk-17-jre` (https://www.ubuntuupdates.org/package/core/lunar/main/base/default-jre) instead? What's wrong with Flink on JDK 17? Worth a comment, at least.